docs/tutorials/build_a_node_in_rust.md

```diff
@@ -111,23 +111,25 @@ let broadcaster = YourTxBroadcaster();
 struct YourPersister();
 
 impl<ChannelSigner: Sign> Persist for YourPersister {
-    fn persist_new_channel(
-        &self, funding_txo: OutPoint, monitor: &ChannelMonitor<ChannelSigner>
+		fn persist_new_channel(
+        &self, 
+        id: OutPoint, 
+        data: &ChannelMonitor<ChannelSigner>
     ) -> Result<(), ChannelMonitorUpdateErr> {
-        // <insert code to persist the ChannelMonitor to disk and/or backups>
-        // Note that monitor.encode() will get you the ChannelMonitor as a
-        // Vec<u8>.
-    }
+			// <insert code to persist the ChannelMonitor to disk and/or backups>
+			// Note that monitor.encode() will get you the ChannelMonitor as a
+			// Vec<u8>.
+		}
 
-	fn update_persisted_channel(
-        &self,
-        funding_txo: OutPoint,
-        update: &ChannelMonitorUpdate,
-        monitor: &ChannelMonitor<ChannelSigner>
+		fn update_persisted_channel(
+        &self, 
+        id: OutPoint, 
+        update: &ChannelMonitorUpdate, 
+        data: &ChannelMonitor<ChannelSigner>
     ) -> Result<(), ChannelMonitorUpdateErr> {
-        // <insert code to persist either the ChannelMonitor or the
-        //  ChannelMonitorUpdate to disk>
-    }
+			// <insert code to persist either the ChannlMonitor or the
+			// ChannelMonitorUpdate to disk>
+		}
```

Comment:
The formatting looks off in the preview. Would you mind updating the example to consistently use spaces? IIRC, the older version of the web site may have treated a tab as four spaces. Or maybe that was what rustdoc does? Can't recall.

FWIW, we use tabs in our code but looks like we are mostly using spaces in these examples.

Reply:
@dennisreimann is possible to set up [prettier](https://prettier.io/) rules for stuff like this?

Reply:
Hmmm... now it is inconsistent with the remaining examples in the file. Could you use spaces?

Sorry, earlier I was just pointing out that our code and the examples in the docs use different formats (tabs and spaces, respectively). But within each we should stick to one format. At very least, each markdown file should be consistent internally.

Reply:
No worries, I actually think my IDE is a bit messed up as I have a bunch of formatting stuff configured from a while ago. 

This any better?